code/game/machinery/iv_drip.dm

```diff
@@ -9,6 +9,21 @@
 	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 	var/obj/item/reagent_containers/iv_bag/bag = null
 
+/obj/machinery/iv_drip/Initialize(mapload)
+	. = ..()
+	if(mapload) // shamelessly stolen locker code
+		addtimer(CALLBACK(src, .proc/take_contents), 0) //  Adding a timer with 0 delay will cause it to only take in contents once the MC has loaded, therefore solving the issue on mapload. During rounds, everything will happen as normal.
```

Comment:
Im questioning if this timer is even needed. All atoms exist before `Initialize()` is called on any of them, so having it just run here without a timer is probably fine.

Also rename `take_contents` to something else. Maybe `check_for_bag`.

Reply:
Proc isnt used anywhere else and the timer isnt needed, the code has been moved to the initialize proc instead in the latest commit. Is this fine?